### ISSUE TITLE
Not able to load pre-trained model (showing tf.loadmodel is not a function)

### ISSUE THREAD

**Issue by kallols** [tensorflow/tfjs#155]
I am a pre-trained model and I want to load it in my js file followed by calling js file into html.

my js code is:

```
async function myFirstTfjs() {
    const model=await tf.loadmodel('./model.json');
        document.getElementById('micro_out_div').innerText += model;
}

myFirstTfjs();

```

my html code is: 

```
<html>
<head>
    <script src="https://cdn.jsdelivr.net/npm/@tensorflow/tfjs@latest"></script>
</head>
<body>
Tiny TFJS example.<hr>
<div id="micro_out_div"></div>
<script src="./index.js"> </script>
</body>
</html>
```

But while running this code  in browser, I am getting the error "tf.loadmodel is not a function" .
please let me know the solution

**Comment by zaidalyafeai** [comment]
I think that should be [loadModal](https://js.tensorflow.org/api/0.8.0/#loadModel) , with capital M.

**Comment by kallols** [comment]
@zaidalyafeai  Thanks, It's working fine with capital M

**Comment by alameen-47** [comment]
Even i got the same problem but not able to understand what actually is the problem is the version mismatch between dependencies or function 
the package.json file as follow of my react native cli app
    "@tensorflow-models/pose-detection": "^2.1.3",
    "@tensorflow/tfjs": "^4.22.0",
    "react-native-tflite": "^0.0.2",
    "@tensorflow/tfjs-data": "^4.22.0",
    "@tensorflow/tfjs-react-native": "^1.0.0",

my function component is as follow that is used to call the model 
import Tflite from 'react-native-tflite';

import Svg, {Circle, Line} from 'react-native-svg';
import {setPose, clearPose} from '../../features/poseSlice.js';
import {useDispatch, useSelector} from 'react-redux';

const {width, height} = Dimensions.get('window');

export const CameraScreenView = () => {
  const navigation = useNavigation();
  const devices = useCameraDevices();
  const device = devices.find(d => d.position === 'back');
  const [isScanning, setIsScanning] = useState(false);
  const [isModelReady, setIsModelReady] = useState(false);
  const cameraRef = useRef(null);
  const detectorRef = useRef(null);
  const dispatch = useDispatch();
  const keypoints = useSelector(state => state.pose.keypoints);
  // const tflite = useRef(new Tflite()).current;
  const tflite = useRef(null);
  useEffect(() => {
    const loadModel = async () => {
      try {
        tflite.current = new Tflite();
        await tflite.current.loadModel({
          model: 'movenet.tflite',
          labels: '',
        });
        setIsModelReady(true);
        console.log('✅ Model loaded successfully');
      } catch (error) {
        console.error('❌ Model loading failed:', error);
      }
    };
    loadModel();
    return () => {
      tflite.current?.close();
    };
  }, []);

I GET THE SAME ERROR AS TypeError:tflite.current.loadModel is not a function (it is undefined )
i have the tf lite model downloaded and placed inside the android/app/src/main/assets/movenet.tflite
HELP ME FIX IT I TRIED MUCH AND I HOPE ITS BECAUSE OF THE VERSION CONFLICTS AND IF POSSIBLE SUGGEST WHICH IS THE BEST COMPATIBLE VERSION FOR ALL THESE TO WORK FINELY FOR A POSEDETECTION MODEL ON REACT NATIVE CLI APP
